RELEASES.md

```diff
@@ -1,3 +1,112 @@
+Version 1.69.0 (2023-04-20)
+==========================
+
+<a id="1.69.0-Language"></a>
+
+Language
+--------
+
+- [Deriving built-in traits on packed structs works with `Copy` fields.](https://github.com/rust-lang/rust/pull/104429/)
+- [Stabilize the `cmpxchg16b` target feature on x86 and x86_64.](https://github.com/rust-lang/rust/pull/106774/)
+- [Add normalization to satisfy trait bounds involving type projections.](https://github.com/rust-lang/rust/pull/103695/)
+- [Add normalization that allows type projections in union fields.](https://github.com/rust-lang/rust/pull/106938/)
```

Comment:
Maybe `Allow associated types to be used as union fields` would be easier to understand? Normalization and projection are more rustc jargon

Reply:
Do you have a suggestion for the prior line as well? Maybe something like "Improve analysis of trait bounds for associated types"?

Reply:
I'm not really familiar with the PR but that sounds reasonable.